lib/src/content/components/main_content/wolt_modal_sheet_main_content.dart

```diff
@@ -82,7 +82,9 @@ class WoltModalSheetMainContent extends StatelessWidget {
               childCount: 2,
             ),
           ),
-        ...page.mainContentSlivers,
+        if (page.mainContentSliversBuilder == null) ...page.mainContentSlivers,
+        if (page.mainContentSliversBuilder != null)
```

Comment:
can we use `else` here to avoid bang operator? Can you also add `assert` to the constructor so that we make sure either one is non-nulll?